dbms/src/Flash/ResourceControl/LocalAdmissionController.cpp

```diff
@@ -0,0 +1,25 @@
+// Copyright 2023 PingCAP, Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <Flash/ResourceControl/LocalAdmissionController.h>
+
+namespace DB
+{
+
+#ifndef DBMS_PUBLIC_GTEST
+std::unique_ptr<LocalAdmissionController> LocalAdmissionController::global_instance;
```

Comment:
So global_instance is null for prod environment?

Reply:
Next pr will update this value. For this pr, `MockLocalAdmissionController` is used

Reply:
https://github.com/pingcap/tiflash/pull/7917/files#diff-6e66cd26a4a06ad28fe982176727d5cf64f7b9c6ed0b167a64dcf68d14642882R30-R32
ResourceGroupQueue seems to be turned on by default, will it cause CI/daily to fail

Reply:
CI is ok, because LocalAdmissionControl::getPriority is simple. So will not coredump even if it's not init.
But it's not reasonable. So i changed it.